scipy/optimize/_minimize.py

```diff
@@ -67,18 +66,19 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
     method : str or callable, optional
         Type of solver.  Should be one of
 
-            - 'Nelder-Mead'
-            - 'Powell'
-            - 'CG'
-            - 'BFGS'
-            - 'Newton-CG'
-            - 'L-BFGS-B'
-            - 'TNC'
-            - 'COBYLA'
-            - 'SLSQP'
-            - 'dogleg'
-            - 'trust-ncg'
-            - custom - a callable object (added in version 0.14.0)
+            - 'Nelder-Mead' :ref:`(see here) <optimize.minimize-neldermead>`
+            - 'Powell'      :ref:`(see here) <optimize.minimize-powell>`
+            - 'CG'          :ref:`(see here) <optimize.minimize-cg>`
+            - 'BFGS'        :ref:`(see here) <optimize.minimize-bfgs>`
+            - 'Newton-CG'   :ref:`(see here) <optimize.minimize-newtoncg>`
+            - 'L-BFGS-B'    :ref:`(see here) <optimize.minimize-lbfgsb>`
+            - 'TNC'         :ref:`(see here) <optimize.minimize-tnc>`
+            - 'COBYLA'      :ref:`(see here) <optimize.minimize-cobyla>`
+            - 'SLSQP'       :ref:`(see here) <optimize.minimize-slsqp>`
+            - 'dogleg'      :ref:`(see here) <optimize.minimize-dogleg>`
+            - 'trust-ncg'   :ref:`(see here) <optimize.minimize-trustncg>`
```

Comment:
Could "reference links" be used instead of "inline" links here? 
```
- `Nelder-Mead`_
[...]

.. _`Nelder-Mead`: optimize.minimize-neldermead

```
That'd be more readable here, and the links could be reused elsewhere in the docstring instead of being repeated.